contracts/ToucanRegenBridge.sol

```diff
@@ -57,16 +59,19 @@ contract ToucanRegenBridge is Ownable, Pausable {
     // ----------------------------------------
 
     /**
-     * @dev Sets the values for {bridgeController} and {toucanContractRegistry}.
+     * @dev Sets the values for {tokenIssuer} and {toucanContractRegistry}.
```

Comment:
just seeing this now, I forgot to mention that the constructor also sets `nctPool` 🤦

Reply:
I think we should just remove the comment as it does not provide much value tbh